spec/graphlient/client_schema_spec.rb

```diff
@@ -1,18 +1,79 @@
 require 'spec_helper'
+require 'tempfile'
 
 describe Graphlient::Client do
-  let(:client) { Graphlient::Client.new('http://graph.biz/graphql') }
+  subject(:client) { described_class.new('http://graph.biz/graphql') }
```

Comment:
It is important to respect existing coding styles. Here `let` calls and explicit constant calls totally work fine. I would rather not make those changes to make the entire diff smaller and easier to understand.

Reply:
can I leave `described_class` part or it should be reverted to?

Reply:
Please do. I personally don't see a reason why this needs to be `described_class`.